server/app.js

```diff
@@ -6,27 +6,19 @@ const bodyParser = require('body-parser');
 
 const path = require('path');
 
-const con = require('./database');
+const fetchAll = require('./model/index.js');
 
 app.use(express.static(path.join(__dirname, '/../client/dist')));
 app.use(bodyParser.json());
 
-
-app.get('/', (req, res) => {
-	console.log('req body is:', req.body);
-	res.send(get());
+app.get('/:listingId/reviews', (req, res) => {
```

Comment:
@gorakong Your endpoint still does not match the one listed in your service plan.

Reply:
@jcelder  whoops! fixed!

Reply:
@gorakong Looks good!